src/settings/AlertWord.js

```diff
@@ -0,0 +1,48 @@
+/* @flow strict-local */
+
+import * as React from 'react';
+import { View, TouchableHighlight, Text, StyleSheet } from 'react-native';
+import { IconCross } from '../common/Icons';
+
+const styles = StyleSheet.create({
+  containerStyle: {
+    backgroundColor: 'white',
+    margin: 5,
+    padding: 5,
+    shadowColor: 'black',
+    shadowOpacity: 0.8,
+    elevation: 6,
+    shadowRadius: 15,
```

Comment:
I'd suggest formatting this style as is done in [this StackOverflow answer](https://stackoverflow.com/a/51302156), to make it clear to the reader what the purpose of these styles are (and that, *e.g.*, changing the `shadowColor` isn't going to have any effect on Android).

... well, assuming this is the UI style we decide to go with. I think it's largely OK as is (possibly modulo some spacing tweaks), but I'm going to call @gnprice in for that.